src/Acme.UI/Models/Athletes/PhysioAthletesViewModel.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Umbraco.Core.Models.PublishedContent;
+using Umbraco.Web.PublishedModels;
+
+namespace Acme.UI.Models.Athletes
+{
+    public class PhysioAthletesViewModel
+    {
+        private List<AthleteViewModel> _athletes;
+
+        private PhysioAthletesViewModel(IPublishedContent containerItem, Physiotherapist physio, int maxAthletesToDisplay)
+        {
+            _athletes = new List<AthleteViewModel>();
+            MaxAthletesToDisplay = maxAthletesToDisplay;
+            if (containerItem != null)
+            {
+                SetUpBaseUrl(containerItem.Url);
+            }
+            SetupPhysioDataData(physio);
+        }
+
+        private void SetUpBaseUrl(string url)
+        {
+            if (url != null)
+            {
+                BaseUrl = url;
+                var lastChar = url.Last();
+
+                if (lastChar != '/' && lastChar != '\\')
+                {
+                    BaseUrl += "/";
+                }
+
+                BaseUrl += "athlete";
+            }
+        }
+
+        private void SetupPhysioDataData(Physiotherapist physio)
+        {
+            if (physio != null)
+            {
+                PhysioName = physio.Physio_name;
+            }
+        }
+
+        public void AddAthlete(AthleteViewModel athlete)
+        {
+            _athletes.Add(athlete);
+        }
+
+        public IReadOnlyList<AthleteViewModel> AllAthletes
+        {
+            get {
+                return _athletes;
+            }        
+        }
+
+        public IReadOnlyList<AthleteViewModel> DisplayableAthletes
+        {
+            get
+            {
+                return MaxAthletesToDisplay > 0 ? _athletes.Take(MaxAthletesToDisplay).ToList() : _athletes;
```

Comment:
What if the take is bigger than the list size

Reply:
Confirmed it will return all the list members.